### ISSUE TITLE
A11y_Model Builder_Add Data_Data Preview_Keyboard : Data Table is not accessed with tab, or any table shortcut keys.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2188]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev1; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG2.1.1; #Keyboard;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Install Visual studio and open Visual studio
2. Navigate to C# project name and right click on it
3. Navigate to "Add" button and press enter
4. Navigate to "Machine learning model" and press enter
5. "Add new item" popup will open
6. Navigate to "Add" button and press enter
7. "Select scenario" popup will open and select "Forecasting" scenario
8. "Select training environment" page will open then select "Next step"
9. "Add Data" page will appear browse and upload the sample data file.
10. tab till Data Preview section.
11. Tab till Table radio button and press enter.
12. Data Table will appear.
13. Check whether Data Table is not accessed with tab, or any table shortcut keys or not.

### Actual Result:
Data Table is not accessed with tab, or any table shortcut keys.

### Expected Result:
Data Table should be accessed with tab, or any table shortcut keys.

![table (1)](https://user-images.githubusercontent.com/91586101/177323119-98e2eb8a-5179-4c98-96ff-274c37f08a14.gif)

**Comment by beccamc** [comment]
@manaswita-chougule  We have previously gotten the feedback that the table should not be tab-able because it is not interactive. 

Please see this issue: https://github.com/dotnet/machinelearning-modelbuilder/issues/1131

**Comment by beccamc** [comment]
Are you in scan mode (caps lock + space)? In scan mode it reads it. 

![scan_mode](https://user-images.githubusercontent.com/9122518/181341005-f4e0bb94-a15f-4678-b2f3-fba5948194ee.gif)

**Comment by manaswita-chougule** [comment]
Closing the bug as the data table is not interactive it is the expected behavior not to be tab-able or accessed by keyboard.
Logged new bug for screen reader issue: https://github.com/dotnet/machinelearning-modelbuilder/issues/2210
Thanks!
